tezos-sig-gating-contracts/scripts/manual/deployNftminter.ts

```diff
@@ -24,7 +24,7 @@ async function main() {
   ledger.set(0, "tz1TiFzFCcwjv4pyYGTrnncqgq17p59CzAE2");
 
   const tokenMetadata = new MichelsonMap();
-  for (let i = 0; i < 10; i++) {
+  for (let i = 0; i < 200; i++) {
```

Comment:
so the limit was increased?

Reply:
I tried to put more , but I reached a limit (transaction size)

Reply:
mmmm I don't understand: this is a script to mint nft initialy right? We don't really need that many. What we need is a high maximum limit for when this example is going to be used o the front end by our customers for demos

Reply:
What   "high maximum limit " stands for 
?

Reply:
the maximum number of nfts that can be minted